x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/core/transform/transforms/TimeRetentionPolicyConfigTests.java

```diff
@@ -34,4 +38,28 @@ protected TimeRetentionPolicyConfig createTestInstance() {
     protected Reader<TimeRetentionPolicyConfig> instanceReader() {
         return TimeRetentionPolicyConfig::new;
     }
+
+    public void testValidationMin() {
+        TimeRetentionPolicyConfig timeRetentionPolicyConfig = new TimeRetentionPolicyConfig(
+            randomAlphaOfLengthBetween(1, 10),
+            TimeValue.timeValueSeconds(10)
+        );
+
+        ActionRequestValidationException e = timeRetentionPolicyConfig.validate(null);
+        assertNotNull(e);
+        assertEquals(1, e.validationErrors().size());
+        assertEquals("retention_policy.time.max_age must be more than 60s, found [10s]", e.validationErrors().get(0));
+    }
+
+    public void testValidationMax() {
+        TimeRetentionPolicyConfig timeRetentionPolicyConfig = new TimeRetentionPolicyConfig(
+            randomAlphaOfLengthBetween(1, 10),
+            TimeValue.parseTimeValue("600000000000d", "time value")
+        );
+
+        ActionRequestValidationException e = timeRetentionPolicyConfig.validate(null);
+        assertNotNull(e);
+        assertEquals(1, e.validationErrors().size());
+        assertEquals("retention_policy.time.max_age must not be greater than [106751.9d]", e.validationErrors().get(0));
```

Comment:
[nit] There is a slight inconsistency between wording of error messages:
"must be more than 60s"
vs
"must not be greater than"